packages/cli/src/databases/migrations/common/1733494526213-AddStatsColumnsToTestRun.ts

```diff
@@ -0,0 +1,11 @@
+import type { MigrationContext, ReversibleMigration } from '@/databases/types';
+
+export class AddStatsColumnsToTestRun1733494526213 implements ReversibleMigration {
+	async up({ schemaBuilder: { addColumns, column } }: MigrationContext) {
+		await addColumns('test_run', [column('total').int, column('passed').int, column('failed').int]);
```

Comment:
Won't this recreate the table in SQLite?

Reply:
Changed this to raw query 👌